go.mod

```diff
@@ -5,7 +5,7 @@ go 1.19
 replace (
 	// dragonberry
 	github.com/confio/ics23/go => github.com/cosmos/cosmos-sdk/ics23/go v0.8.0
-	github.com/cosmos/cosmos-sdk => github.com/scrtlabs/cosmos-sdk v0.45.13-0.20230726163133-dc2b5eab1db5
+	github.com/cosmos/cosmos-sdk => github.com/scrtlabs/cosmos-sdk v0.45.13-0.20230808154448-1e7ad22811a4
```

Comment:
Seeing this https://github.com/scrtlabs/cosmos-sdk/commit/1e7ad22811a4, is there a way to pass `height-1` when we call this? I'm not sure what are the performance implications.

Reply:
If we pass height-1 it breaks all other queries other than wasm

Reply:
wasm doesn't care as it has the cache

Reply:
I told Itzik about this, we thought maybe we should see what happens and if there are performance implications, we know what to look for. Also memory implications, as now queries have the whole db. But probably none of these implications exist as these are all pointers to subsections of the iavl store?

Reply:
If we see performance issues, we could probably even create a context with both `height` and `height - 1`

Reply:
I tried to find a way to do this but it seems that the multiStore there didn't support more than one specific version, or all of them